tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py

```diff
@@ -270,7 +270,7 @@
                 Field(
                     item="32A",
                     name='FED_OASDI_PROGRAM',
-                    friendly_name="federal old-age survivors and disability insurance program",
```

Comment:
The codebook's reference is: Receives Federal Disability Insurance Benefits Under Title II of the Social Security Act (Social Security Disability Insurance [SSDI]).